doc/ansible/dev/developer_guide.md

```diff
@@ -61,7 +66,7 @@ we will create and delete a namespace with the switch of a variable:
 ```yaml
 ---
 - name: set example-memcached namespace to {{ state }}
-  k8s:
+  community.kubernetes.k8s:
```

Comment:
(Same notes as above on the collections keyword... I just hate seeing FQCNs everywhere, Ansible starts looking like Java...)